src/js/views/contactsView.js

```diff
@@ -1,10 +1,10 @@
 import {elements} from './base';
 
-export const renderContact = (doc) => {
+export const renderContact = (doc,profilePic) => {
     const markup = `
```

Comment:
Can we have null check here too like https://github.com/Yuxuan-Lin/VioLegacy/blob/be77883d0471ff00803f314eb656118676c71045/src/js/views/chatView.js#L9?

Reply:
Done